src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,15 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => {
+  const { user, title, completed } = todo;
+
+  return (
+    <article className={`TodoInfo ${completed && 'TodoInfo--completed'}`}>
```

Comment:
It's better to use [classnames](https://www.npmjs.com/package/classnames) library to set them conditionally.
Because now incompleted articles looks like
![image](https://github.com/mate-academy/react_static-list-of-todos-js/assets/63243136/2a8e9123-77c4-4299-af7b-b0c3d07f30d4)